app/upgrades/v0.0.12.go

```diff
@@ -0,0 +1,154 @@
+package upgrades
+
+import (
+	"context"
+
+	"cosmossdk.io/math"
+	storetypes "cosmossdk.io/store/types"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	cosmosTypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/pokt-network/poktroll/app/keepers"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+)
+
+const Upgrade_0_0_12_PlanName = "v0.0.12"
+
+// Upgrade_0_0_12 handles the upgrade to release `v0.0.12`. To be issued on both Alpha and Beta TestNets.
+var Upgrade_0_0_12 = Upgrade{
+	PlanName: Upgrade_0_0_12_PlanName,
+	CreateUpgradeHandler: func(mm *module.Manager,
+		keepers *keepers.Keepers,
+		configurator module.Configurator,
+	) upgradetypes.UpgradeHandler {
+		// Parameter configurations aligned with repository config.yml specifications
+		// These values reflect the delta between v0.0.11 and current main branch
+		// Reference:
+		// - Comparison: https://github.com/pokt-network/poktroll/compare/v0.0.11..main
```

Comment:
Should we be using `main` or a specific sha (that equals to main) when you're preparing this PR?

That way this will be "evergreen"

Reply:
Moving conversation to discord to get this over the finish line.

https://discord.com/channels/824324475256438814/1332065342746525778/1336434350660784239